CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Releases
 
+## v1.5.0 (November 4, 2019)
+
+### Enhancements
+
+- **Deep Dependency Graph:** Implement Service Oriented Dependency Graph (DDG) ([@tiffon](https://github.com/tiffon) in [#442](https://github.com/jaegertracing/jaeger-ui/pull/442), [#441](https://github.com/jaegertracing/jaeger-ui/pull/441), [#440](https://github.com/jaegertracing/jaeger-ui/pull/440), [#439](https://github.com/jaegertracing/jaeger-ui/pull/439), [#438](https://github.com/jaegertracing/jaeger-ui/pull/438), [#437](https://github.com/jaegertracing/jaeger-ui/pull/437), [#431](https://github.com/jaegertracing/jaeger-ui/pull/431), [#406](https://github.com/jaegertracing/jaeger-ui/pull/406), and [#405](https://github.com/jaegertracing/jaeger-ui/pull/405); and [@everett980](https://github.com/everett980) in [#468](https://github.com/jaegertracing/jaeger-ui/pull/468), [#466](https://github.com/jaegertracing/jaeger-ui/pull/466), [#458](https://github.com/jaegertracing/jaeger-ui/pull/458), [#463](https://github.com/jaegertracing/jaeger-ui/pull/463), [#457](https://github.com/jaegertracing/jaeger-ui/pull/457), [#453](https://github.com/jaegertracing/jaeger-ui/pull/453), [#452](https://github.com/jaegertracing/jaeger-ui/pull/452), [#434](https://github.com/jaegertracing/jaeger-ui/pull/434), [#433](https://github.com/jaegertracing/jaeger-ui/pull/433), [#432](https://github.com/jaegertracing/jaeger-ui/pull/432), [#429](https://github.com/jaegertracing/jaeger-ui/pull/429), [#427](https://github.com/jaegertracing/jaeger-ui/pull/427), [#424](https://github.com/jaegertracing/jaeger-ui/pull/424), [#421](https://github.com/jaegertracing/jaeger-ui/pull/421), [#412](https://github.com/jaegertracing/jaeger-ui/pull/412), [#398](https://github.com/jaegertracing/jaeger-ui/pull/398), [#386](https://github.com/jaegertracing/jaeger-ui/pull/386), and [#378](https://github.com/jaegertracing/jaeger-ui/pull/378))
```

Comment:
could we just book an issue and reference it here, and the linkage to PRs can be from the issue itself. If I am a user reading this CL, I probably don't care about every commit that went into implementing the feature.

Reply:
also, the title says "improve DDG", could probably use a bit more description if there's some significant improvement

Reply:
yeah I didn't love this approach, but felt it was better than listing every PR and didn't want to lose the tie between the `CHANGELOG.md` and the PRs if every wanted.

an intermediary issue is even better.

Reply:
you can simply move this list of diffs to an issue